src/components/BnListbox/BnListbox.styles.cjs

```diff
@@ -39,5 +48,11 @@ module.exports = {
         '@apply hover:bg-varColor-600 ui-selected:bg-varColor-600': {},
       },
     },
+    '&__option-text': {
+      '@apply truncate': {},
+    },
+    '&__selected-option-icon': {
+      '@apply ml-auto h-3.5 w-4 shrink-0': {},
```

Comment:
I wonder if they arbitrary value would work here, since it'd be parsed directly 🤔

Reply:
mmm it might, but I don't think it makes much of a difference here